executor/cleanup.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package poolmgr
+package executor
```

Comment:
Hmm, this _is_ a poolmgr-specific file. How about we make a subdirectory under here, and move the poolmgr into it?